tests/test_graph.py

```diff
@@ -0,0 +1,17 @@
+from starcli.draw_graph import draw_graph,draw_top_rep_graphs
+
+def test_draw_graph():
+    reponame = 'hedyhli/starcli'
+    auth = 'tongjin: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'
```

Comment:
Never commit your tokens to a public repository! You should delete the token as soon as possible.

You can use `--auth` to supply this when running pytest, see the "Running tests" section of [CONTRIBUTING.md](https://github.com/hedyhli/starcli/blob/main/CONTRIBUTING.md).